src/TestSuite/Fixture/TestFixture.php

```diff
@@ -100,7 +101,7 @@ public function __construct()
                 $message = sprintf(
                     'Invalid datasource name "%s" for "%s" fixture. Fixture datasource names must begin with "test".',
```

Comment:
Maybe $this->name / $this->table to be changed to class_name($this) or sth to make it more correct?